inlong-tubemq/pom.xml

```diff
@@ -41,6 +41,7 @@
 
     <properties>
         <inlong.root.dir>${project.parent.basedir}</inlong.root.dir>
+        <jcommander.version>1.78</jcommander.version>
```

Comment:
There is `jcommander.version` definition in the pom.xml, and you could use it directly.

Reply:
I will delete this line soon.